n3fit/src/n3fit/backends/keras_backend/MetaModel.py

```diff
@@ -42,18 +45,20 @@
     "SGD": (Kopt.SGD, {"learning_rate": 0.01, "momentum": 0.0, "nesterov": False}),
 }
 
+NN = "NN"
+preprocessing = "preprocessing_factor"
```

Comment:
```suggestion
NN_PREFIX = "NN"
PREPROCESSING_PREFIX = "preprocessing_factor"
```

(and the associated changes in the rest of the file)